src/CLI/ScanDelay.cs

```diff
@@ -25,35 +27,35 @@ public void DelayWithCountdown(IOptions options)
 
             if (options.DelayInSeconds > 0)
             {
-                ConditionallyWriteMessageWithCount(options, "Delaying {0} second{1} before scanning.", options.DelayInSeconds);
+                ConditionallyWriteMessageWithCount(options, DisplayStrings.ScanDelayHeaderOneSecond,
+                    DisplayStrings.ScanDelayHeaderMoreThanOneSecond, options.DelayInSeconds);
                 for (int secondsRemaining = options.DelayInSeconds; secondsRemaining > 0; secondsRemaining--)
                 {
-                    ConditionallyWriteMessageWithCount(options, "  {0} second{1} before scan.", secondsRemaining);
+                    ConditionallyWriteMessageWithCount(options, DisplayStrings.ScanDelayCountdownOneSecondLeft,
+                        DisplayStrings.ScanDelayCountdownMoreThanOneSecondLeft, secondsRemaining);
                     _oneSecondDelay.Invoke();
                 }
-                ConditionallyWriteMessage(options, "Triggering scan");
+                ConditionallyWriteMessage(options, () => DisplayStrings.ScanDelayTriggeringScan);
             }
         }
 
-        private void ConditionallyWriteMessageWithCount(IOptions options, string format, int count)
+        private void ConditionallyWriteMessageWithCount(IOptions options, string stringIfCountIsOne, string stringIfCountIsNotOne, int count)
         {
-            if (options.VerbosityLevel > VerbosityLevel.Quiet)
-            {
-                _writer.WriteLine(format, count, PluralSuffix(count));
-            }
+            string format = SelectString(stringIfCountIsOne, stringIfCountIsNotOne, count);
+            ConditionallyWriteMessage(options, () => string.Format(CultureInfo.InvariantCulture, format, count));
         }
 
-        private void ConditionallyWriteMessage(IOptions options, string message)
+        private void ConditionallyWriteMessage(IOptions options, Func<string> getMessage)
```

Comment:
This now takes a Func so that we can consolidate the options check in one place but still only build the strings if we really use them. This last bit helps the unit tests to better validate the behavior.